src/client/app/containers/BarChartContainer.ts

```diff
@@ -78,7 +78,7 @@ function mapStateToProps(state: State) {
 				const hoverText: string[] = [];
 				const readings = _.orderBy(readingsData.readings, ['startTimestamp'], ['asc']);
 				readings.forEach(barReading => {
-					// calculate more precise bar duration 
+					// subtracting one extra day caused by day ending at midnight of the next day
```

Comment:
@huss how do I remove the change from the PR? Shall I go revert the comment and make a new commit? when I kept the old comment it said merge conflict because of this comment

Reply:
I believe the bigger issue here is that your branch is not up to date with the development branch on the OED GitHub account. Thus, it sees your comment (before the change) as changing the same line as something already in development so it sees a merge conflict. I just tried updating your PR with development and the change was no longer listed. While I can push my changes to your PR, it is probably best if you do this so your workflow will be okay in the future. There are lots of websites that describe how to do this but I think these are the basic steps you need to follow:

1. Get your clone of your fork of OED on your local machine to know about the OED repo. If you do `git remote -v` on your local machine in your clone of OED, I suspect you will see only your account on GitHub listed as origin (and not the OED repo on that account as upstream). If so, do `git remote add upstream https://github.com/OpenEnergyDashboard/OED.git`. Now if you do `git remote -v` you should see your the OED repo as upstream.
2. Get the changes from upstream (OED repo) into your work. There are several ways to do this but `git fetch` should work fine. (You can also pull from the OED repo.) There can be merge conflicts but should not be in this case. Now, you won't see your comment as a change. In the future, you can avoid the edit by doing this so the change will come from development branch on OED. Also note that you must push this out to get your GitHub account (origin) synced with your local clone that was just changed.

As I was finishing this I looked and it seems you have already brought development into your PR so you probably did something like this. I'm just leaving this for the record and you can comment if my steps are off.